README.md

```diff
@@ -130,6 +130,12 @@ logging:
   # regardless of the configured logging level.
   sampled_levels:
     'trace/webrequest': 0.2
+  # Set up the limiter for logging. When the rate of logs is close to the limit,
+  # it starts to increase the probability of discarding the logs of same kind.
+  # For default parameters use rate_limit: true
+  rate_limit:
```

Comment:
You are right though that the matching semantics are different. The normal level selector semantics are "this level and up", while sampled_levels has no such implication, so can be used to target specific lower-level events.

I was wondering though if it would make sense to unify them under the hood, and perhaps even at a higher level. One (strawman) way of doing so could perhaps be something like this:

```yaml
level_rates: 
  >=warn: 200
  trace/webrequest: 10
```

This would set a target rate limit of 200/s for warnings & higher, and 10/s for trace/webrequest.